packages/no-case/src/index.ts

```diff
@@ -17,12 +19,16 @@ const DEFAULT_STRIP_REGEXP = /[^A-Z0-9]+/gi;
  * Normalize the string into something other libraries can manipulate easier.
  */
 export function noCase(input: string, options: Options = {}) {
-  const {
-    splitRegexp = DEFAULT_SPLIT_REGEXP,
+  let {
```

Comment:
Prefer to keep `const`, you can either destructure it later to re-use earlier variables or just make it a new constant in a separate statement. No reason to use `let` here.

Reply:
Yeah makes sense. Following the latter conversation, I'll resolve this by appending the new regexes to the regex list instead of making a new list.